### PR TITLE
Pre model version 1.3

### PR DESCRIPTION
**Pull request by lxatrhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/COMET-SDK-Community-Edition/pulls) open
- [x] I have verified that I am following the COMET-SDK [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/COMET-SDK-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description
Some changes that were necessary according to the first model version 1.3 try-out

### CODE REVIEW

File: CDP4Common/CDP4Common.csproj [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <PackageTags>CDP COMET ECSS-E-TM-10-25</PackageTags>
     <PackageLicenseExpression>LGPL-3.0-only</PackageLicenseExpression>
     <PackageReleaseNotes>
-		[Fix] Bugfix NestedElement.GetChildren; fixes #171
+		[Upgrade] Model version 1.3
```
Comment (samatrhea) [inline CDP4Common/CDP4Common.csproj:22]: i doubt that this is the case, please revert

File: CDP4Dal/CDP4Dal.csproj [hunk 2]
```diff
@@ -19,8 +19,7 @@
     <PackageTags>CDP COMET ECSS-E-TM-10-25</PackageTags>
     <PackageLicenseExpression>LGPL-3.0-only</PackageLicenseExpression>
     <PackageReleaseNotes>
-      [Fix] Concurrency issues in Session; fixes #167
-      [Improve] Memory consumption CDPMessageBus; fixes #176
+      [Upgrade] Model version 1.3
```
Comment (samatrhea) [inline CDP4Dal/CDP4Dal.csproj:22]: revert

File: CDP4JsonSerializer/CDP4JsonSerializer.csproj [hunk 3]
```diff
@@ -19,8 +19,7 @@
     <PackageTags>CDP COMET ECSS-E-TM-10-25 JSON</PackageTags>
     <PackageLicenseExpression>LGPL-3.0-only</PackageLicenseExpression>
     <PackageReleaseNotes>
-      [Update] to CDP4Common 7.4.0
-      [Fix] Copy ElementDefinition between models fails if model version is different #184
+      [Upgrade] Model version 1.3
```
Comment (samatrhea) [inline CDP4JsonSerializer/CDP4JsonSerializer.csproj:22]: revert

File: CDP4Rules/CDP4Rules.csproj [hunk 4]
```diff
@@ -19,7 +19,7 @@
     <PackageTags>CDP COMET ECSS-E-TM-10-25</PackageTags>
     <PackageLicenseExpression>LGPL-3.0-only</PackageLicenseExpression>
     <PackageReleaseNotes>
-      [Update] to CDP4Common 7.4.0
+      [Upgrade] Model version 1.3
```
Comment (samatrhea) [inline CDP4Rules/CDP4Rules.csproj:22]: revert

File: CDP4ServicesDal/CDP4ServicesDal.csproj [hunk 5]
```diff
@@ -19,7 +19,7 @@
     <PackageTags>CDP COMET ECSS-E-TM-10-25</PackageTags>
     <PackageLicenseExpression>LGPL-3.0-only</PackageLicenseExpression>
     <PackageReleaseNotes>
-      [Update] to CDP4Dal 7.4.0
+      [Upgrade] Model version 1.3
```
Comment (samatrhea) [inline CDP4ServicesDal/CDP4ServicesDal.csproj:22]: revert

File: CDP4WspDal/CDP4WspDal.csproj [hunk 6]
```diff
@@ -19,7 +19,7 @@
     <PackageTags>CDP COMET ECSS-E-TM-10-25</PackageTags>
     <PackageLicenseExpression>LGPL-3.0-only</PackageLicenseExpression>
     <PackageReleaseNotes>
-      [Update] to CDP4Dal 7.4.0
+      [Upgrade] Model version 1.3
```
Comment (samatrhea) [inline CDP4WspDal/CDP4WspDal.csproj:22]: revert

**Review by samatrhea — CHANGES_REQUESTED** [review]
I notice that you have removed the warning on the generated files that any manual changes would be removed after a new generation of said files. I propose we keep that notice to make that clear to developers.

Also, the PackageReleaseNotes state an upgrade to model version 1.3 which i think is to soon

**Review by alexatrhea — APPROVED** [review]

**Review by samatrhea — APPROVED** [review]

**Pull request merged**